runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorkerTest.java

```diff
@@ -2725,14 +2725,14 @@ public void testActiveWorkForShardedKeys() throws Exception {
 
     // Verify a different shard of key is a separate queue.
     Work m4 = createMockWork(3);
-    assertFalse(computationState.activateWork(key1Shard1, m4));
+    assertTrue(computationState.activateWork(key1Shard1, m4));
```

Comment:
why did this change? it seems like it should be duplicate of m3.  Think this might be related to early return I commented on in code.